docs/index.md

```diff
@@ -4,4 +4,4 @@ title: Home
 description: Home page for the Pulsar Editor
 ---
 
-# Hello Vuepress
+# Hello World
```

Comment:
Funny, I'll get to finishing this later. I have a plan for this index and just need to get around to it. XD

Reply:
Aha, yeah sorry that was just to test that I was able to make the changes in the right place, as this was my first time messing with Vuepress